drush/rbsc-example.aliases.drushrc.php

```diff
@@ -17,31 +17,31 @@
  * Production alias
  * Set each option to match your configuration
  */
-// $aliases['prod'] = array (
-//   'uri' => 'https://rbsc.princeton.edu',
-//   'root' => '', // Add root
-//   'remote-user' => '', // Add user
-//   'remote-host' => '', // Add host
-//   'ssh-options' => '-o PasswordAuthentication=no -i .ssh/id_rsa', // Add ssh
-//   'path-aliases' => array(
-//     '%dump-dir' => '/tmp',
-//     '%files' => 'sites/default/files',
-//   ),
-//   'source-command-specific' => array (
-//     'sql-sync' => array (
-//       'no-cache' => TRUE,
-//       'structure-tables-key' => 'common',
-//     ),
-//   ),
-//   'command-specific' => array (
-//     'sql-sync' => array (
-//       'sanitize' => TRUE,
-//       'no-ordered-dump' => TRUE,
-//       'structure-tables' => array(
-//        // You can add more tables which contain data to be ignored by the database dump
-//         'common' => array('cache', 'cache_filter', 'cache_menu', 'cache_page', 'history', 'sessions', 'watchdog'),
-//       ),
-//     ),
-//   ),
-// );
+$aliases['prod'] = array (
+  'uri' => 'https://rbsc.princeton.edu',
+  'root' => '', // Add root
+  'remote-user' => '', // Add user
+  'remote-host' => 'lib-sc-prod1', // Add host
+  'ssh-options' => '-o PasswordAuthentication=no -i .ssh/id_rsa', // Add ssh
+  'path-aliases' => array(
+    '%dump-dir' => '/tmp',
+    '%files' => 'sites/default/files',
+  ),
+  'source-command-specific' => array (
+    'sql-sync' => array (
+      'no-cache' => TRUE,
+      'structure-tables-key' => 'common',
+    ),
+  ),
+  'command-specific' => array (
+    'sql-sync' => array (
+      'sanitize' => TRUE,
+      'no-ordered-dump' => TRUE,
+      'structure-tables' => array(
+       // You can add more tables which contain data to be ignored by the database dump
+        'common' => array('cache', 'cache_filter', 'cache_menu', 'cache_page', 'history', 'sessions', 'watchdog'),
+      ),
+    ),
+  ),
+);
```

Comment:
@carolyncole I approved it but I dont know why this part is uncommented. I'm sure you know.